authors/apps/articles/views.py

```diff
@@ -0,0 +1,195 @@
+from django.shortcuts import render
```

Comment:
Hello @a-braham ,
You are not using this module, so you don't need to import it